src/v2/spec.yaml

```diff
@@ -1982,6 +2013,17 @@ paths:
           example: 0xa6a8736f18f383f1cc2d938576933e5ea7df01a1,0x7cac817861e5c3384753403fb6c0c556c204b1ce
           schema:
             type: string
+        - name: address_types
+          description: Customize which address types the request should search for. Default is all. Select multiple types by using a comma-separated list of valid types in the enum
```

Comment:
isn't this a different PR?

Reply:
yeah I saw that - i think thats below, not sure why its showing, should have been up to date 🤔  @rishavmukherji 

https://github.com/neynarxyz/OAS/pull/51